### ISSUE TITLE
Working directly with cache

### ISSUE THREAD

**Issue by andrejkaurin** [FacultyCreative/ngActiveResource#42]
What about working and querying on cache explicitly? Sometimes there are needs to work in "offline" mode and then sync data after some period of time.

Update: 
- all query methods should also be available for cache
- cached.$save should be also available with some flag like $transient=true

**Comment by brayniverse** [comment]
@andrejkaurin That's cool, its something I needed for a previous project. Would be cool to see it here too.

**Comment by jamescam22** [comment]
Yeah, this library is pretty much a no-go for me since I'm developing on PhoneGap... Offline mode is pretty imperative!
